src/twisted/runner/procmon.py

```diff
@@ -103,14 +128,26 @@ class ProcessMonitor(service.Service):
     def __init__(self, reactor=_reactor):
         self._reactor = reactor
 
-        self.processes = {}
+        self._processes = {}
         self.protocols = {}
         self.delay = {}
         self.timeStarted = {}
         self.murder = {}
         self.restart = {}
 
 
+    @deprecate.deprecatedProperty(incremental.Version("Twisted", 17, 10, 0))
+    def processes(self):
+        """
+        Processes as list of tuples
+
+        @return: List of monitored processes as tuples
+        """
+        return {name: process.toTuple()
+                for name, process in self._processes.items()}
+
+
+    @deprecate.deprecated(incremental.Version("Twisted", 17, 10, 0))
```

Comment:
why are we deprecating pickle support? just asking.

I don't see any reference in the release notes about this deprecation